core/storage/exploration/gae_models.py

```diff
@@ -402,6 +403,27 @@ def delete_multi( # type: ignore[override]
             ExplorationCommitLogEntryModel.update_timestamps_multi(
                 commit_log_models)
             datastore_services.put_multi(commit_log_models)
+        else:
+            # Delete the ExplorationVersionHistoryModels if force_deletion is
+            # True.
+            if force_deletion:
+                versioned_models_without_none = [
+                    model for model in versioned_models if model is not None]
```

Comment:
Under what circumstances would it be None?

Reply:
It is actually the exploration models, hence, I have changed the name now to `versioned_exp_models` and `versioned_exp_models_without_none` to avoid confusion.